### PR TITLE
secondlife/viewer#1333 BOOL to bool conversion leftovers

### PR DESCRIPTION
**Pull request by LLGuru**

### CODE REVIEW

File: indra/llmath/llsphere.cpp [hunk 1]
```diff
@@ -93,9 +93,8 @@ F32 LLSphere::getOverlap(const LLSphere& other_sphere) const
 
 bool LLSphere::operator==(const LLSphere& rhs) const
 {
-	// TODO? -- use approximate equality for centers?
-	return (mRadius == rhs.mRadius
-			&& mCenter == rhs.mCenter);
+	return fabs(mRadius - rhs.mRadius) <= FLT_EPSILON &&
```
Comment (akleshchev) [inline indra/llmath/llsphere.cpp:96]: Are you sure nothing in code relies on old behavior?

File: scripts/messages/message_template.msg.sha1 [hunk 2]
```diff
@@ -1 +1 @@
-d7915d67467e59287857630bd89bf9529d065198
+d7915d67467e59287857630bd89bf9529d065199
```
Comment (marchcat) [inline scripts/messages/message_template.msg.sha1:1]: This file is not needed here.

**Review by akleshchev — APPROVED** [review]

File: indra/llmath/llsphere.cpp (hunk 1 above)
Comment (akleshchev) [reply indra/llmath/llsphere.cpp:96]: Never mind, LLSphere appears to be unused

File: indra/newview/llimview.cpp [hunk 3]
```diff
@@ -206,51 +206,51 @@ void notify_of_message(const LLSD& msg, bool is_dnd_msg)
...
         else
         {
         	user_preferences = gSavedSettings.getString("NotificationNonFriendIMOptions");
-			if (!gAgent.isDoNotDisturb() && (gSavedSettings.getBOOL("PlaySoundNonFriendIM") == true))
+			if (!gAgent.isDoNotDisturb() && (gSavedSettings.getBOOL("PlaySoundNonFriendIM")))
 			{
 				make_ui_sound("UISndNewIncomingIMSession");
-        }
-    }
+			}
```
Comment (marchcat) [inline indra/newview/llimview.cpp:239]: These will be changed back to spaces soon.

File: scripts/messages/message_template.msg.sha1 (hunk 2 above)
Comment (LLGuru) [reply scripts/messages/message_template.msg.sha1]: > This file is not needed here.

Removed, thanks

File: indra/newview/llimview.cpp (hunk 3 above)
Comment (marchcat) [reply indra/newview/llimview.cpp:239]: Already done :)
https://github.com/secondlife/viewer/blob/e7eced3c87310b15ac20cc3cd470d67686104a14/scripts/code_tools/fix_whitespace.py#L43

**Review by marchcat — APPROVED** [review]

**Pull request merged**